tests/odov3/go.mod

```diff
@@ -3,69 +3,77 @@ module github.com/devfile/registry/tests/odov3
 go 1.18
 
 require (
-	github.com/devfile/api/v2 v2.0.0-20220309195345-48ebbf1e51cf
+	github.com/devfile/api/v2 v2.2.0
 	github.com/devfile/library v1.2.1-0.20220602130922-85a4805bd59c
-	github.com/onsi/ginkgo/v2 v2.5.0
-	github.com/onsi/gomega v1.24.0
+	github.com/onsi/ginkgo/v2 v2.13.0
```

Comment:
Ginkgo usually recommends that the Ginkgo CLI and library versions be kept in sync. Otherwise, it can lead to weird issues. Ginkgo detects this and displays warnings, like [here](https://github.com/devfile/registry/actions/runs/6733416790/job/18302055330?pr=238#step:6:14).
So I'd suggest using the same `v2.13.0` version everywhere we install the Ginkgo CLI, like [here](https://github.com/devfile/registry/blob/main/.github/workflows/validate-stacks.yaml#L51) (and potentially other places).

Reply:
Nice Catch! Thnx! I've updated the workflows that they were installing ginkgo